admin_guide/install/first_steps.adoc

```diff
@@ -6,73 +6,177 @@
-
-The example applications are all built out of GitHub repositories which are referenced by the templates by default, as seen in the SOURCE_REPOSITORY_URL parameter value.  You can fork those repositories and provide your own SOURCE_REPOSITORY_URL parameter value when creating from the template to experiment with creating your own applications.
-
-
+- The link:../docker_registry.html[integrated docker
+registry] service must be deployed in your OpenShift installation.
+- You must be able to run the following CLI commands with
+link:../../architecture/additional_concepts/authorization.html#roles[*cluster-admin*
+privileges], because they operate on the default *openshift*
+link:../../architecture/core_objects/openshift_model.html#project[project].
+
+== Creating Image Streams for OpenShift Images
+The core set of image streams provide images that can be used to build
+link:../../using_images/sti_images/nodejs.html[*Node.js*],
+link:../../using_images/sti_images/perl.html[*Perl*],
+link:../../using_images/sti_images/php.html[*PHP*],
+link:../../using_images/sti_images/python.html[*Python*], and
+link:../../using_images/sti_images/ruby.html[*Ruby*] applications. It also
+defines images for link:../../using_images/db_images/mongodb.html[*MongoDB*],
+link:../../using_images/db_images/mysql.html[*MySQL*], and
+link:../../using_images/db_images/postgresql.html[*PostgreSQL*] to support data
+storage.
+
+To create the core set of image streams, which use CentOS 7 based images:
+
+----
+$ oc create -f \
+    examples/image-streams/image-streams-centos7.json \
+    -n openshift
+----
+
+Alternatively, if you are using node hosts that are subscribed using Red Hat
+Subscription Management and prefer to use the Red Hat Enterprise Linux (RHEL) 7
+based images:
+
+----
+$ oc create -f \
+    examples/image-streams/image-streams-rhel7.json \
+    -n openshift
+----
+
+It is not possible to create both the CentOS and RHEL sets of image streams
+because they use the same names. If you desire to have both sets of image
+streams available to users, either create one set in a different project, or
+edit one of the files and modify the image stream names to make them unique.
+
+ifdef::openshift-enterprise[]
+== Creating Image Streams for xPaaS Middleware Images
+The xPaaS Middleware image streams provide images for
+link:../../using_images/xpaas_images/eap.html[*JBoss EAP*],
+link:../../using_images/xpaas_images/jws.html[*JBoss EWS*], and
+link:../../using_images/xpaas_images/a_mq.html[*JBoss A-MQ*]. They can be used
+to build applications for those platforms or run directly as is.
+
+To create the xPaaS Middleware set of image streams:
+
+----
+$ oc create -f \
+    examples/middleware-streams/jboss-image-streams.json
```

Comment:
we might as well change the dirname to xpaas-streams while we're at it.  this dir doesn't actually exist yet.  @sdodson @brenton